scripts/create_all_packages.sh

```diff
@@ -80,6 +80,13 @@ if [ "${DRYRUN}zz" == "zz" ]; then
     # Change permissions on source files to match those expected by the server
     chmod 644 build/*
     CURRENT_DATE=$(date +'%Y-%m-%d')
+    if [ ! "${CRON_LAUNCHED}zz" == "zz" ]; then
```

Comment:
It certainly works, but I don't find this line easy to understand.

It's my fault, it's the code style I had used in this script. I'm not proud of that: I just hope I had a good reason, at that time, like execution on very old versions of bash because of Travis?

Anyway, there are more explicit ways to test if a variable exists and is not empty.
Try with:
```suggestion
    if [ -n "${CRON_LAUNCHED}" ]; then
```
(the same syntax could be used above, and maybe in other scripts)
Feel free to move that in a separate issue if you wish